src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsCcdDataSource.java

```diff
@@ -2,25 +2,35 @@
 
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections4.CollectionUtils;
 import uk.gov.hmcts.ecm.common.client.CcdClient;
-import uk.gov.hmcts.ecm.common.model.reports.hearingstojudgments.HearingsToJudgmentsSubmitEvent;
 import uk.gov.hmcts.ethos.replacement.docmosis.reports.ReportException;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @RequiredArgsConstructor
 @Slf4j
-public class HearingsToJudgmentsCcdReportDataSource implements HearingsToJudgmentsReportDataSource {
+public class HearingsToJudgmentsCcdDataSource implements HearingsToJudgmentsDataSource {
 
     private final String authToken;
     private final CcdClient ccdClient;
 
     @Override
-    public List<HearingsToJudgmentsSubmitEvent> getData(String caseTypeId,
+    public List<HearingsToJudgmentsSubmitEvent> getData(String caseTypeId, String managingOffice,
                                                         String listingDateFrom, String listingDateTo) {
+
         try {
-            var query = HearingsToJudgmentsElasticSearchQuery.create(listingDateFrom, listingDateTo);
-            return ccdClient.hearingsToJudgementsSearch(authToken, caseTypeId, query);
+            var query = HearingsToJudgmentsElasticSearchQuery.create(managingOffice, listingDateFrom, listingDateTo);
+            var submitEvents = new ArrayList<HearingsToJudgmentsSubmitEvent>();
+            var searchResult = ccdClient.runElasticSearch(authToken, caseTypeId, query,
+                    HearingsToJudgmentsSearchResult.class);
+
+            if (searchResult != null && CollectionUtils.isNotEmpty(searchResult.getCases())) {
+                submitEvents.addAll(searchResult.getCases());
```

Comment:
May be just return searchResult.getCases() without having to declare submitEvents?

Reply:
refactored to just return the case list rather than setting another variable